monai/data/image_reader.py

```diff
@@ -107,47 +110,48 @@ def verify_suffix(self, filename: Union[Sequence[str], str]) -> bool:
         """
         return True
 
-    def read(self, data: Union[Sequence[str], str, Any]):
+    def read(self, data: Union[Sequence[str], str], **kwargs):
         """
-        Read image data from specified file or files, or set a `itk.Image` object.
+        Read image data from specified file or files.
         Note that the returned object is ITK image object or list of ITK image objects.
-        `self._img` is always a list, even only has 1 image.
 
         Args:
-            data: file name or a list of file names to read, or a `itk.Image` object for the usage that
-                the image data is already in memory and no need to read from file again.
+            data: file name or a list of file names to read,
+            kwargs: additional args for `itk.imread` API, if not set, using `self.kwargs`.
+                More details about available args:
+                https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itkExtras.py
 
         """
-        self._img = list()
-        if isinstance(data, Image):
-            self._img.append(data)
-            return data
+        img_: List[Image] = list()
 
         filenames: Sequence[str] = ensure_tuple(data)
+        if len(kwargs) == 0:
+            kwargs = self.kwargs
         for name in filenames:
-            self._img.append(itk.imread(name, **self.kwargs))
-        return self._img if len(filenames) > 1 else self._img[0]
+            img_.append(itk.imread(name, **kwargs))
```

Comment:
would be great to have a test case with some non-empty `kwargs`

Reply:
added in the latest commit.